pydy/tests/test_system.py

```diff
@@ -395,3 +397,27 @@ def test_integrate(self):
         sys = System(self.kane, times=times)
         with testing.assert_raises(NotImplementedError):
             sys.generate_ode_function(generator='made-up')
+
+
+class TestJointSystem():
+
+    def setup(self):
+        self.system = System.using_joints()
+
+    def test_init(self):
+        assert isinstance(self.system.ground, Ground)
+        assert isinstance(self.system.origin, Point)
+        assert isinstance(self.system.reference_frame, ReferenceFrame)
+        assert self.system.bodies == list()
+        assert self.system.is_using_joints is True
+
+    def test_gravity(self):
+        assert self.system.gravitational_constant == sm.Symbol('gravity')
```

Comment:
The user should be able to optionally set this symbol and it should also be able to be set to zero. I'd make the default simply `'g'`.